tests/manage/z_cluster/nodes/test_nodes_restart.py

```diff
@@ -60,17 +60,25 @@ def test_nodes_restart(self, nodes, pvc_factory, pod_factory, force):
         self.sanity_helpers.create_resources(pvc_factory, pod_factory)
 
     @bugzilla('1754287')
+    @bugzilla('1873938')
     @pytest.mark.polarion_id("OCS-2015")
     def test_rolling_nodes_restart(self, nodes, pvc_factory, pod_factory):
         """
         Test restart nodes one after the other and check health status in between
 
         """
         ocp_nodes = get_node_objs()
+        pv_current = get_pv_names()
+        pv_list = []
```

Comment:
the validation that unexpected local PV doesn't exist is relevant to LSO deployments only